lib/shimmer.js

```diff
@@ -387,19 +387,16 @@ const shimmer = (module.exports = {
   setupSubscribers: function setupSubscribers(agent) {
     this._subscribers = {}
     for (const subscriberConfig of Object.values(subscriptions)) {
-      const Subscriber = require(`./subscribers/${subscriberConfig.path}`)
-      const subscriber = new Subscriber({ agent, logger })
-      if (subscriber.enabled === false) {
-        logger.debug(
-          'Skipping subscriber %s because it is disabled in the config',
-          subscriber.id
-        )
-        continue
+      if (Array.isArray(subscriberConfig)) {
```

Comment:
i would change all the configurations to be an array that way there's one way to initialize subscribers.